[TensorExpr] Fix imports in tensorexpr benchmarks.

ZolotukhinM opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* **#35830 [TensorExpr] Fix imports in tensorexpr benchmarks.**

Differential Revision: [D20799464](https://our.internmc.facebook.com/intern/diff/D20799464)

Commit:
[TensorExpr] Fix imports in tensorexpr benchmarks.

[ghstack-poisoned]

Commit:
Update on "[TensorExpr] Fix imports in tensorexpr benchmarks."


Differential Revision: [D20799464](https://our.internmc.facebook.com/intern/diff/D20799464)

[ghstack-poisoned]

zheng-xq commented on benchmarks/tensorexpr/__main__.py:
```diff
@@ -4,14 +4,16 @@
-# import conv
-# import matmul
+from . import attention      # noqa: F401
+from . import broadcast      # noqa: F401
```
I think only "broadcast" and "elementwise" work for now. But if nothing is causing problems so far, I am fine with that.

ZolotukhinM replied:
Some other work too, but some of them don't - you are right. If we don't run them, then don't make any troubles AFAIU.

Commit:
Update on "[TensorExpr] Fix imports in tensorexpr benchmarks."


Differential Revision: [D20799464](https://our.internmc.facebook.com/intern/diff/D20799464)

[ghstack-poisoned]

Commit:
Update on "[TensorExpr] Fix imports in tensorexpr benchmarks."


Differential Revision: [D20799464](https://our.internmc.facebook.com/intern/diff/D20799464)

[ghstack-poisoned]

Commit:
Update on "[TensorExpr] Fix imports in tensorexpr benchmarks."


Differential Revision: [D20799464](https://our.internmc.facebook.com/intern/diff/D20799464)

[ghstack-poisoned]
